File: WorldFeed/Server/src/WorldFeed.Common/Models/Properties/QuickReplyOption.cs
namespace WorldFeed.Common.Models.Properties
{
    using Newtonsoft.Json;

    using WorldFeed.Common.Public.Models.Interfaces;

    public class QuickReplyOption : IQuickReplyOption
    {
        [JsonProperty("label")]
        public string Label { get; set; }

        [JsonProperty("description")]
        public string Description { get; set; }

        [JsonProperty("metadata")]
        public string Metadata { get; set; }
    }
}

File: Sharebook/Client/src/app/core/Core/Models/Properties/QuickReplyOption.ts
import {IQuickReplyOption} from "../../../Public/Models/Interfaces/IQuickReplyOption";

export class QuickReplyOption implements IQuickReplyOption {
  // [JsonProperty("label")]
  public label: string;

  // [JsonProperty("description")]
  public description: string;

  // [JsonProperty("metadata")]
  public metadata: string;
}
